tests/test_transport_network.py

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python3
+
+
+import pathlib
+
+import pytest  # noqa: F401
+
+import r5py
+import com.conveyal.r5
+import java.time
+
+DATA_DIRECTORY = pathlib.Path(__file__).absolute().parent.parent / "docs" / "data"
+OSM_PBF = DATA_DIRECTORY / "kantakaupunki.osm.pbf"
+GTFS = DATA_DIRECTORY / "GTFS.zip"
+
+GTFS_TIMEZONE = "Europe/Helsinki"
+# TRANSPORT_NETWORK_CHECKSUM = 925935595  # not consistent between computers
+
+
+class Test_TransportNetwork:
+    @pytest.fixture(scope="session")
+    def transport_network_from_test_files(self):
+        transport_network = r5py.TransportNetwork(OSM_PBF, [GTFS])
+        return transport_network
+
+    @pytest.fixture(scope="session")
+    def transport_network_from_test_directory(self):
+        return r5py.TransportNetwork.from_directory(DATA_DIRECTORY)
+
+    def test_init_from_files_and_dir_cover_same_extent(
+        self, transport_network_from_test_files, transport_network_from_test_directory
+    ):
+        assert (
+            transport_network_from_test_files._transport_network.getEnvelope()
+            == transport_network_from_test_directory._transport_network.getEnvelope()
+        )
+        # assert (
```

Comment:
@christophfink Are these comments important? Wouldn't want to merge these kind of unused codeblocks to main, so perhaps clean up from here?

Reply:
I thought of leaving these here for reference, to remember that this precisely does not work, but a similar approach might be good

Reply:
Hmm, could you rather e.g make an issue or add a comment to current issue with these lines of codes for reminder? I have a bit annoying experiences cleaning these kind of "I just leave these here for now" kind of lines in the past 😆 We have made good efforts keeping the codebase clean, and would like to continue doing so 🙂

Reply:
Absolutely, I'll convert them into comments or an issue (let's see how I feel in the morning)